pinot-broker/src/main/java/org/apache/pinot/broker/broker/BasicAuthAccessControlFactory.java

```diff
@@ -46,17 +47,14 @@
  *     pinot.broker.access.control.principals.user456.tables=stuff,lessImportantStuff
  * </pre>
  */
+@NoArgsConstructor
```

Comment:
I don't think this is needed because there is no other constructor. Same for other classes with single empty constructor

Reply:
make sense. will clean up